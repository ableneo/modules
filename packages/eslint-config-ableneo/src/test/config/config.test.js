import {CLIEngine} from "eslint";
import path from "path";

describe("config", () => {
  it("full eslintrc.json", () => {
    const fixturePath = path.resolve(__dirname, "fixtures", "utils.js");
    const cliEngine = new CLIEngine({});

    expect({
      ...cliEngine.getConfigForFile(fixturePath),
      parser: "[parser]",
    }).toMatchSnapshot();
  });
});
