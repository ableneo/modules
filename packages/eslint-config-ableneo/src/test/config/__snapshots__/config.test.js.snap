// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config full eslintrc.json 1`] = `
Object {
  "env": Object {
    "browser": true,
    "commonjs": true,
    "es6": true,
    "jest": true,
    "jest/globals": true,
    "node": true,
  },
  "globals": Object {},
  "ignorePatterns": Array [],
  "noInlineConfig": undefined,
  "parser": "[parser]",
  "parserOptions": Object {
    "ecmaFeatures": Object {
      "jsx": true,
    },
    "ecmaVersion": 2018,
    "sourceType": "module",
  },
  "plugins": Array [
    "prettier",
    "jest",
    "@typescript-eslint",
    "react-hooks",
    "jsx-a11y",
    "flowtype",
    "import",
    "react",
  ],
  "reportUnusedDisableDirectives": undefined,
  "rules": Object {
    "@typescript-eslint/adjacent-overload-signatures": Array [
      "error",
    ],
    "@typescript-eslint/ban-ts-ignore": Array [
      "error",
    ],
    "@typescript-eslint/ban-types": Array [
      "error",
    ],
    "@typescript-eslint/brace-style": Array [
      "off",
    ],
    "@typescript-eslint/camelcase": Array [
      "error",
    ],
    "@typescript-eslint/class-name-casing": Array [
      "error",
    ],
    "@typescript-eslint/comma-spacing": Array [
      "off",
    ],
    "@typescript-eslint/consistent-type-assertions": Array [
      "error",
    ],
    "@typescript-eslint/explicit-function-return-type": Array [
      "off",
    ],
    "@typescript-eslint/explicit-member-accessibility": Array [
      "off",
    ],
    "@typescript-eslint/func-call-spacing": Array [
      "off",
    ],
    "@typescript-eslint/indent": Array [
      "off",
    ],
    "@typescript-eslint/interface-name-prefix": Array [
      "error",
    ],
    "@typescript-eslint/member-delimiter-style": Array [
      "off",
    ],
    "@typescript-eslint/no-array-constructor": Array [
      "error",
    ],
    "@typescript-eslint/no-empty-function": Array [
      "off",
    ],
    "@typescript-eslint/no-empty-interface": Array [
      "error",
    ],
    "@typescript-eslint/no-explicit-any": Array [
      "off",
    ],
    "@typescript-eslint/no-extra-parens": Array [
      "off",
    ],
    "@typescript-eslint/no-extra-semi": Array [
      "off",
    ],
    "@typescript-eslint/no-inferrable-types": Array [
      "error",
    ],
    "@typescript-eslint/no-misused-new": Array [
      "error",
    ],
    "@typescript-eslint/no-namespace": Array [
      "error",
    ],
    "@typescript-eslint/no-non-null-assertion": Array [
      "warn",
    ],
    "@typescript-eslint/no-object-literal-type-assertion": Array [
      "off",
    ],
    "@typescript-eslint/no-this-alias": Array [
      "error",
    ],
    "@typescript-eslint/no-unused-vars": Array [
      "off",
    ],
    "@typescript-eslint/no-use-before-define": Array [
      "off",
      Object {
        "classes": false,
        "functions": false,
        "typedefs": false,
        "variables": false,
      },
    ],
    "@typescript-eslint/no-var-requires": Array [
      "off",
    ],
    "@typescript-eslint/prefer-namespace-keyword": Array [
      "error",
    ],
    "@typescript-eslint/quotes": Array [
      0,
    ],
    "@typescript-eslint/semi": Array [
      "off",
    ],
    "@typescript-eslint/space-before-function-paren": Array [
      "off",
    ],
    "@typescript-eslint/triple-slash-reference": Array [
      "error",
    ],
    "@typescript-eslint/type-annotation-spacing": Array [
      "off",
    ],
    "array-bracket-newline": Array [
      "off",
    ],
    "array-bracket-spacing": Array [
      "off",
    ],
    "array-callback-return": Array [
      "warn",
    ],
    "array-element-newline": Array [
      "off",
    ],
    "arrow-body-style": Array [
      0,
    ],
    "arrow-parens": Array [
      "off",
    ],
    "arrow-spacing": Array [
      "off",
    ],
    "block-spacing": Array [
      "off",
    ],
    "brace-style": Array [
      "off",
    ],
    "camelcase": Array [
      "off",
    ],
    "comma-dangle": Array [
      "off",
    ],
    "comma-spacing": Array [
      "off",
    ],
    "comma-style": Array [
      "off",
    ],
    "computed-property-spacing": Array [
      "off",
    ],
    "constructor-super": Array [
      "error",
    ],
    "curly": Array [
      0,
    ],
    "default-case": Array [
      "warn",
      Object {
        "commentPattern": "^no default$",
      },
    ],
    "dot-location": Array [
      "off",
      "property",
    ],
    "eol-last": Array [
      "off",
    ],
    "eqeqeq": Array [
      "warn",
      "smart",
    ],
    "flowtype/boolean-style": Array [
      "off",
      "boolean",
    ],
    "flowtype/define-flow-type": Array [
      "warn",
    ],
    "flowtype/delimiter-dangle": Array [
      "off",
    ],
    "flowtype/generic-spacing": Array [
      "off",
      "never",
    ],
    "flowtype/no-mixed": Array [
      0,
    ],
    "flowtype/no-types-missing-file-annotation": Array [
      2,
    ],
    "flowtype/no-weak-types": Array [
      0,
    ],
    "flowtype/object-type-delimiter": Array [
      "off",
    ],
    "flowtype/require-parameter-type": Array [
      0,
    ],
    "flowtype/require-readonly-react-props": Array [
      0,
    ],
    "flowtype/require-return-type": Array [
      0,
    ],
    "flowtype/require-valid-file-annotation": Array [
      "warn",
    ],
    "flowtype/semi": Array [
      "off",
    ],
    "flowtype/space-after-type-colon": Array [
      "off",
      "always",
    ],
    "flowtype/space-before-generic-bracket": Array [
      "off",
      "never",
    ],
    "flowtype/space-before-type-colon": Array [
      "off",
      "never",
    ],
    "flowtype/type-id-match": Array [
      0,
    ],
    "flowtype/union-intersection-spacing": Array [
      "off",
      "always",
    ],
    "flowtype/use-flow-type": Array [
      "warn",
    ],
    "flowtype/valid-syntax": Array [
      "off",
    ],
    "for-direction": Array [
      "error",
    ],
    "func-call-spacing": Array [
      "off",
    ],
    "function-call-argument-newline": Array [
      "off",
    ],
    "function-paren-newline": Array [
      "off",
    ],
    "generator-star": Array [
      "off",
    ],
    "generator-star-spacing": Array [
      "off",
    ],
    "getter-return": Array [
      "warn",
    ],
    "implicit-arrow-linebreak": Array [
      "off",
    ],
    "import/first": Array [
      "error",
    ],
    "import/newline-after-import": Array [
      "error",
    ],
    "import/no-absolute-path": Array [
      "error",
    ],
    "import/no-amd": Array [
      "error",
    ],
    "import/no-extraneous-dependencies": Array [
      "error",
      Object {
        "devDependencies": Array [
          "**/test/**",
          "**/tests/**",
          "**/spec/**",
          "**/__tests__/**",
          "**/__mocks__/**",
          "**/test.{js,jsx,ts,tsx}",
          "test-*.{js,jsx,ts,tsx}",
          "**/*{.,_}{test,spec}.{js,jsx,ts,tsx}",
          "**/jest.config.js",
          "**/jest.setup.js",
          "**/vue.config.js",
          "**/webpack.config.js",
          "**/webpack.config.*.js",
          "**/rollup.config.js",
          "**/rollup.config.*.js",
        ],
        "optionalDependencies": false,
      },
    ],
    "import/no-webpack-loader-syntax": Array [
      "error",
    ],
    "import/prefer-default-export": Array [
      "off",
    ],
    "indent": Array [
      "off",
    ],
    "indent-legacy": Array [
      "off",
    ],
    "jest/expect-expect": Array [
      "warn",
    ],
    "jest/no-commented-out-tests": Array [
      "warn",
    ],
    "jest/no-disabled-tests": Array [
      "warn",
    ],
    "jest/no-export": Array [
      "error",
    ],
    "jest/no-focused-tests": Array [
      "error",
    ],
    "jest/no-identical-title": Array [
      "error",
    ],
    "jest/no-jasmine-globals": Array [
      "warn",
    ],
    "jest/no-jest-import": Array [
      "error",
    ],
    "jest/no-mocks-import": Array [
      "error",
    ],
    "jest/no-standalone-expect": Array [
      "error",
    ],
    "jest/no-test-callback": Array [
      "error",
    ],
    "jest/no-test-prefixes": Array [
      "error",
    ],
    "jest/no-try-expect": Array [
      "error",
    ],
    "jest/valid-describe": Array [
      "error",
    ],
    "jest/valid-expect": Array [
      "error",
    ],
    "jest/valid-expect-in-promise": Array [
      "error",
    ],
    "jsx-a11y/accessible-emoji": Array [
      "warn",
    ],
    "jsx-a11y/alt-text": Array [
      "warn",
    ],
    "jsx-a11y/anchor-has-content": Array [
      "warn",
    ],
    "jsx-a11y/anchor-is-valid": Array [
      "warn",
      Object {
        "aspects": Array [
          "noHref",
          "invalidHref",
        ],
      },
    ],
    "jsx-a11y/aria-activedescendant-has-tabindex": Array [
      "warn",
    ],
    "jsx-a11y/aria-props": Array [
      "warn",
    ],
    "jsx-a11y/aria-proptypes": Array [
      "warn",
    ],
    "jsx-a11y/aria-role": Array [
      "warn",
      Object {
        "ignoreNonDOM": true,
      },
    ],
    "jsx-a11y/aria-unsupported-elements": Array [
      "warn",
    ],
    "jsx-a11y/heading-has-content": Array [
      "warn",
    ],
    "jsx-a11y/href-no-hash": Array [
      "off",
    ],
    "jsx-a11y/iframe-has-title": Array [
      "warn",
    ],
    "jsx-a11y/img-redundant-alt": Array [
      "warn",
    ],
    "jsx-a11y/no-access-key": Array [
      "warn",
    ],
    "jsx-a11y/no-distracting-elements": Array [
      "warn",
    ],
    "jsx-a11y/no-redundant-roles": Array [
      "warn",
    ],
    "jsx-a11y/role-has-required-aria-props": Array [
      "warn",
    ],
    "jsx-a11y/role-supports-aria-props": Array [
      "warn",
    ],
    "jsx-a11y/scope": Array [
      "warn",
    ],
    "jsx-quotes": Array [
      "off",
    ],
    "key-spacing": Array [
      "off",
    ],
    "keyword-spacing": Array [
      "off",
    ],
    "linebreak-style": Array [
      "off",
    ],
    "lines-around-comment": Array [
      0,
    ],
    "max-len": Array [
      0,
    ],
    "multiline-ternary": Array [
      "off",
    ],
    "new-parens": Array [
      "off",
    ],
    "newline-per-chained-call": Array [
      "off",
    ],
    "no-array-constructor": Array [
      "warn",
    ],
    "no-arrow-condition": Array [
      "off",
    ],
    "no-async-promise-executor": Array [
      "error",
    ],
    "no-caller": Array [
      "warn",
    ],
    "no-case-declarations": Array [
      "error",
    ],
    "no-class-assign": Array [
      "error",
    ],
    "no-comma-dangle": Array [
      "off",
    ],
    "no-compare-neg-zero": Array [
      "error",
    ],
    "no-cond-assign": Array [
      "warn",
      "except-parens",
    ],
    "no-confusing-arrow": Array [
      0,
    ],
    "no-console": Array [
      "warn",
    ],
    "no-const-assign": Array [
      "warn",
    ],
    "no-constant-condition": Array [
      "error",
    ],
    "no-control-regex": Array [
      "warn",
    ],
    "no-debugger": Array [
      "warn",
    ],
    "no-delete-var": Array [
      "warn",
    ],
    "no-dupe-args": Array [
      "warn",
    ],
    "no-dupe-class-members": Array [
      "warn",
    ],
    "no-dupe-keys": Array [
      "warn",
    ],
    "no-duplicate-case": Array [
      "warn",
    ],
    "no-empty": Array [
      "error",
      Object {
        "allowEmptyCatch": true,
      },
    ],
    "no-empty-character-class": Array [
      "warn",
    ],
    "no-empty-function": Array [
      "off",
    ],
    "no-empty-pattern": Array [
      "warn",
    ],
    "no-eval": Array [
      "warn",
    ],
    "no-ex-assign": Array [
      "warn",
    ],
    "no-extend-native": Array [
      "warn",
    ],
    "no-extra-bind": Array [
      "warn",
    ],
    "no-extra-boolean-cast": Array [
      "error",
    ],
    "no-extra-label": Array [
      "warn",
    ],
    "no-extra-parens": Array [
      "off",
    ],
    "no-extra-semi": Array [
      "off",
    ],
    "no-fallthrough": Array [
      "warn",
    ],
    "no-floating-decimal": Array [
      "off",
    ],
    "no-func-assign": Array [
      "warn",
    ],
    "no-global-assign": Array [
      "error",
    ],
    "no-implied-eval": Array [
      "warn",
    ],
    "no-inner-declarations": Array [
      "error",
    ],
    "no-invalid-regexp": Array [
      "warn",
    ],
    "no-irregular-whitespace": Array [
      "off",
    ],
    "no-iterator": Array [
      "warn",
    ],
    "no-label-var": Array [
      "warn",
    ],
    "no-labels": Array [
      "warn",
      Object {
        "allowLoop": true,
        "allowSwitch": false,
      },
    ],
    "no-lone-blocks": Array [
      "warn",
    ],
    "no-loop-func": Array [
      "warn",
    ],
    "no-misleading-character-class": Array [
      "error",
    ],
    "no-mixed-operators": Array [
      0,
      Object {
        "allowSamePrecedence": false,
        "groups": Array [
          Array [
            "&",
            "|",
            "^",
            "~",
            "<<",
            ">>",
            ">>>",
          ],
          Array [
            "==",
            "!=",
            "===",
            "!==",
            ">",
            ">=",
            "<",
            "<=",
          ],
          Array [
            "&&",
            "||",
          ],
          Array [
            "in",
            "instanceof",
          ],
        ],
      },
    ],
    "no-mixed-spaces-and-tabs": Array [
      "off",
    ],
    "no-multi-spaces": Array [
      "off",
    ],
    "no-multi-str": Array [
      "warn",
    ],
    "no-multiple-empty-lines": Array [
      "off",
    ],
    "no-native-reassign": Array [
      "off",
    ],
    "no-negated-in-lhs": Array [
      "off",
    ],
    "no-new-func": Array [
      "warn",
    ],
    "no-new-object": Array [
      "warn",
    ],
    "no-new-symbol": Array [
      "warn",
    ],
    "no-new-wrappers": Array [
      "warn",
    ],
    "no-obj-calls": Array [
      "warn",
    ],
    "no-octal": Array [
      "warn",
    ],
    "no-octal-escape": Array [
      "warn",
    ],
    "no-prototype-builtins": Array [
      "error",
    ],
    "no-redeclare": Array [
      "warn",
      Object {
        "builtinGlobals": false,
      },
    ],
    "no-regex-spaces": Array [
      "warn",
    ],
    "no-reserved-keys": Array [
      "off",
    ],
    "no-restricted-globals": Array [
      "error",
      "addEventListener",
      "blur",
      "close",
      "closed",
      "confirm",
      "defaultStatus",
      "defaultstatus",
      "event",
      "external",
      "find",
      "focus",
      "frameElement",
      "frames",
      "history",
      "innerHeight",
      "innerWidth",
      "length",
      "location",
      "locationbar",
      "menubar",
      "moveBy",
      "moveTo",
      "name",
      "onblur",
      "onerror",
      "onfocus",
      "onload",
      "onresize",
      "onunload",
      "open",
      "opener",
      "opera",
      "outerHeight",
      "outerWidth",
      "pageXOffset",
      "pageYOffset",
      "parent",
      "print",
      "removeEventListener",
      "resizeBy",
      "resizeTo",
      "screen",
      "screenLeft",
      "screenTop",
      "screenX",
      "screenY",
      "scroll",
      "scrollbars",
      "scrollBy",
      "scrollTo",
      "scrollX",
      "scrollY",
      "self",
      "status",
      "statusbar",
      "stop",
      "toolbar",
      "top",
    ],
    "no-restricted-properties": Array [
      "error",
      Object {
        "message": "Please use import() instead. More info: https://facebook.github.io/create-react-app/docs/code-splitting",
        "object": "require",
        "property": "ensure",
      },
      Object {
        "message": "Please use import() instead. More info: https://facebook.github.io/create-react-app/docs/code-splitting",
        "object": "System",
        "property": "import",
      },
    ],
    "no-restricted-syntax": Array [
      "warn",
      "WithStatement",
    ],
    "no-script-url": Array [
      "warn",
    ],
    "no-self-assign": Array [
      "warn",
    ],
    "no-self-compare": Array [
      "warn",
    ],
    "no-sequences": Array [
      "warn",
    ],
    "no-shadow-restricted-names": Array [
      "warn",
    ],
    "no-space-before-semi": Array [
      "off",
    ],
    "no-spaced-func": Array [
      "off",
    ],
    "no-sparse-arrays": Array [
      "warn",
    ],
    "no-tabs": Array [
      0,
    ],
    "no-template-curly-in-string": Array [
      "warn",
    ],
    "no-this-before-super": Array [
      "warn",
    ],
    "no-throw-literal": Array [
      "warn",
    ],
    "no-trailing-spaces": Array [
      "off",
    ],
    "no-undef": Array [
      "error",
    ],
    "no-unexpected-multiline": Array [
      0,
    ],
    "no-unreachable": Array [
      "warn",
    ],
    "no-unsafe-finally": Array [
      "error",
    ],
    "no-unsafe-negation": Array [
      "error",
    ],
    "no-unused-expressions": Array [
      "error",
      Object {
        "allowShortCircuit": true,
        "allowTaggedTemplates": true,
        "allowTernary": true,
      },
    ],
    "no-unused-labels": Array [
      "warn",
    ],
    "no-unused-vars": Array [
      "warn",
      Object {
        "args": "none",
        "ignoreRestSiblings": true,
      },
    ],
    "no-use-before-define": Array [
      "warn",
      Object {
        "classes": false,
        "functions": false,
        "variables": false,
      },
    ],
    "no-useless-catch": Array [
      "error",
    ],
    "no-useless-computed-key": Array [
      "warn",
    ],
    "no-useless-concat": Array [
      "warn",
    ],
    "no-useless-constructor": Array [
      "warn",
    ],
    "no-useless-escape": Array [
      "warn",
    ],
    "no-useless-rename": Array [
      "warn",
      Object {
        "ignoreDestructuring": false,
        "ignoreExport": false,
        "ignoreImport": false,
      },
    ],
    "no-var": Array [
      "error",
    ],
    "no-whitespace-before-property": Array [
      "off",
    ],
    "no-with": Array [
      "warn",
    ],
    "no-wrap-func": Array [
      "off",
    ],
    "nonblock-statement-body-position": Array [
      "off",
    ],
    "object-curly-newline": Array [
      "off",
    ],
    "object-curly-spacing": Array [
      "off",
    ],
    "object-property-newline": Array [
      "off",
    ],
    "one-var-declaration-per-line": Array [
      "off",
    ],
    "operator-linebreak": Array [
      "off",
    ],
    "padded-blocks": Array [
      "off",
    ],
    "padding-line-between-statements": Array [
      "error",
      Object {
        "blankLine": "always",
        "next": "return",
        "prev": "*",
      },
      Object {
        "blankLine": "always",
        "next": "*",
        "prev": Array [
          "const",
          "let",
          "var",
        ],
      },
      Object {
        "blankLine": "any",
        "next": Array [
          "const",
          "let",
          "var",
        ],
        "prev": Array [
          "const",
          "let",
          "var",
        ],
      },
    ],
    "prefer-arrow-callback": Array [
      0,
    ],
    "prefer-const": Array [
      "error",
    ],
    "prefer-rest-params": Array [
      "error",
    ],
    "prefer-spread": Array [
      "error",
    ],
    "prettier/prettier": Array [
      "error",
      Object {
        "bracketSpacing": false,
        "trailingComma": "all",
      },
    ],
    "quote-props": Array [
      "off",
    ],
    "quotes": Array [
      0,
    ],
    "react-hooks/exhaustive-deps": Array [
      "warn",
    ],
    "react-hooks/rules-of-hooks": Array [
      "error",
    ],
    "react/display-name": Array [
      "off",
    ],
    "react/forbid-foreign-prop-types": Array [
      "warn",
      Object {
        "allowInPropTypes": true,
      },
    ],
    "react/jsx-child-element-spacing": Array [
      "off",
    ],
    "react/jsx-closing-bracket-location": Array [
      "off",
    ],
    "react/jsx-closing-tag-location": Array [
      "off",
    ],
    "react/jsx-curly-newline": Array [
      "off",
    ],
    "react/jsx-curly-spacing": Array [
      "off",
    ],
    "react/jsx-equals-spacing": Array [
      "off",
    ],
    "react/jsx-filename-extension": Array [
      "off",
    ],
    "react/jsx-first-prop-new-line": Array [
      "off",
    ],
    "react/jsx-indent": Array [
      "off",
    ],
    "react/jsx-indent-props": Array [
      "off",
    ],
    "react/jsx-key": Array [
      2,
    ],
    "react/jsx-max-props-per-line": Array [
      "off",
    ],
    "react/jsx-no-comment-textnodes": Array [
      2,
    ],
    "react/jsx-no-duplicate-props": Array [
      2,
    ],
    "react/jsx-no-target-blank": Array [
      2,
    ],
    "react/jsx-no-undef": Array [
      2,
    ],
    "react/jsx-one-expression-per-line": Array [
      "off",
    ],
    "react/jsx-pascal-case": Array [
      "warn",
      Object {
        "allowAllCaps": true,
        "ignore": Array [],
      },
    ],
    "react/jsx-props-no-multi-spaces": Array [
      "off",
    ],
    "react/jsx-space-before-closing": Array [
      "off",
    ],
    "react/jsx-tag-spacing": Array [
      "off",
    ],
    "react/jsx-uses-react": Array [
      2,
    ],
    "react/jsx-uses-vars": Array [
      2,
    ],
    "react/jsx-wrap-multilines": Array [
      "off",
    ],
    "react/no-children-prop": Array [
      2,
    ],
    "react/no-danger-with-children": Array [
      2,
    ],
    "react/no-deprecated": Array [
      2,
    ],
    "react/no-direct-mutation-state": Array [
      2,
    ],
    "react/no-find-dom-node": Array [
      2,
    ],
    "react/no-is-mounted": Array [
      2,
    ],
    "react/no-render-return-value": Array [
      2,
    ],
    "react/no-string-refs": Array [
      2,
    ],
    "react/no-typos": Array [
      "error",
    ],
    "react/no-unescaped-entities": Array [
      2,
    ],
    "react/no-unknown-property": Array [
      2,
    ],
    "react/no-unsafe": Array [
      0,
    ],
    "react/prefer-stateless-function": Array [
      "warn",
    ],
    "react/prop-types": Array [
      "off",
    ],
    "react/react-in-jsx-scope": Array [
      2,
    ],
    "react/require-render-return": Array [
      2,
    ],
    "react/style-prop-object": Array [
      "warn",
    ],
    "require-yield": Array [
      "warn",
    ],
    "rest-spread-spacing": Array [
      "off",
      "never",
    ],
    "semi": Array [
      "off",
    ],
    "semi-spacing": Array [
      "off",
    ],
    "semi-style": Array [
      "off",
    ],
    "space-after-function-name": Array [
      "off",
    ],
    "space-after-keywords": Array [
      "off",
    ],
    "space-before-blocks": Array [
      "off",
    ],
    "space-before-function-paren": Array [
      "off",
    ],
    "space-before-function-parentheses": Array [
      "off",
    ],
    "space-before-keywords": Array [
      "off",
    ],
    "space-in-brackets": Array [
      "off",
    ],
    "space-in-parens": Array [
      "off",
    ],
    "space-infix-ops": Array [
      "off",
    ],
    "space-return-throw-case": Array [
      "off",
    ],
    "space-unary-ops": Array [
      "off",
    ],
    "space-unary-word-ops": Array [
      "off",
    ],
    "strict": Array [
      "warn",
      "never",
    ],
    "switch-colon-spacing": Array [
      "off",
    ],
    "template-curly-spacing": Array [
      "off",
    ],
    "template-tag-spacing": Array [
      "off",
    ],
    "unicode-bom": Array [
      "off",
      "never",
    ],
    "use-isnan": Array [
      "warn",
    ],
    "valid-typeof": Array [
      "warn",
    ],
    "wrap-iife": Array [
      "off",
    ],
    "wrap-regex": Array [
      "off",
    ],
    "yield-star-spacing": Array [
      "off",
    ],
    "yoda": Array [
      "error",
      "never",
      Object {
        "exceptRange": true,
        "onlyEquality": false,
      },
    ],
  },
  "settings": Object {
    "flowtype": Object {
      "onlyFilesWithFlowAnnotation": false,
    },
    "import/ignore": Array [
      "node_modules",
      "\\\\.(coffee|scss|css|less|hbs|svg|json)$",
    ],
    "import/resolver": Object {
      "node": Object {
        "extensions": Array [
          ".mjs",
          ".js",
          ".json",
        ],
      },
    },
    "react": Object {
      "version": "detect",
    },
  },
}
`;
